src/runtime/pkg/katautils/config.go

```diff
@@ -1564,6 +1578,11 @@ func checkFactoryConfig(config oci.RuntimeConfig) error {
 // checkHypervisorConfig performs basic "sanity checks" on the hypervisor
 // config.
 func checkHypervisorConfig(config vc.HypervisorConfig) error {
+
+	if config.RemoteHypervisorSocket != "" {
```

Comment:
Would make sense to have some silly checks like "timeout is unsigned int greater than zero"?

Reply:
OK. I'll add such checks.

Reply:
RemoteHypervisorSocket is an unsigned variable, and zero means no timoeout, so we don't need to non-negative check here.

Reply:
Ok, makes sense. Thanks for addressing my comment!